src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTaggerProvider.cs

```diff
@@ -57,13 +57,13 @@ internal class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdornmentTag
             _classificationTypeRegistryService = classificationTypeRegistryService;
         }
 
-        protected internal override bool SupportsDiagnosticMode(DiagnosticMode mode)
+        protected sealed override bool SupportsDiagnosticMode(DiagnosticMode mode)
```

Comment:
technically redundant since the type is sealed, but i find this much easier to reason about when just looking at the member.